docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.4'
+services:
+  mariadb:
+    image: mariadb:latest
+    environment:
+      MYSQL_ROOT_PASSWORD: changeme
+      MYSQL_DATABASE: yotter
+      MYSQL_USER: yotter
+      MYSQL_PASSWORD: changeme
+    restart: on-failure
+    volumes:
+     - /var/lib/mysql
+    healthcheck:
```

Comment:
Remove that. It just delays the time until the container is ready in a single container setup.

Reply:
Remove what? The restart policy or the health check?